pkg/algorithm/hived_algorithm_test.go

```diff
@@ -509,6 +592,34 @@ func testSuggestedNodes(t *testing.T, h *HivedAlgorithm) {
 	compareSchedulingResult(t, pod, psr)
 }
 
+func testStatefulPreemption(t *testing.T, configFilePath string) {
+	sConfig := api.NewConfig(api.InitRawConfig(&configFilePath))
```

Comment:
Also add test for preemption cancel/undo?

Reply:
see pod30